pkg/apis/core/types.go

```diff
@@ -2638,6 +2638,13 @@ type PodSecurityContext struct {
 	// for that container.
 	// +optional
 	RunAsUser *int64
+	// The GID to run the entrypoint of the container process.
+	// Defaults to group specified in image metadata if unspecified.
+	// May also be set in SecurityContext.  If set in both SecurityContext and
+	// PodSecurityContext, the value specified in SecurityContext takes precedence
+	// for that container.
```

Comment:
Nit: you can strike the 'for that container' wording and just end the sentence at 'precedence'.

Reply:
fixed